.github/workflows/update-infrastructure-docker.yaml

```diff
@@ -0,0 +1,29 @@
+name: Update infrastructure-docker submodule
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  schema:
+    name: Update infrastructure-docker submodule
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: "misarch/infrastructure-docker"
+          submodules: true
+      - name: Update submodule
+        run: |
+          cd catalog
+          git checkout ${{ github.sha }}
```

Comment:
Is the `git push` performed by `create PR`?

Reply:
yes